client/game/commands.py

```diff
@@ -30,7 +31,10 @@ def __init__(self, profile, queue, position):
 class RequestSendChat(Command):
     def __init__(self, profile, queue, message):
         super().__init__(profile, queue, f"Request sending the chat message:{message}")
-        self.events = [SendChatRequestEvent(message)]
+        self.events = [
+            SendChatRequestEvent(message),
+            ChatMessageInGameEvent(profile.id, message),
```

Comment:
The chat message creation on the ui now happens before the server has confirmed it